index.html

```diff
@@ -173,5 +262,56 @@ <h4 style="font-weight: lighter;">We know how large objects will act, but things
             </div>
             </div>
         </section>
+
+        <section class="why-us">
+        <div class="why-choose-us">
+            <h2 class="why-choose-us-head">WHY CHOOSE US</h2>
```

Comment:
use CSS to make the text uppercase. Don't write it in uppercase in HTML as it is not a flexible solution.